src/forms.ts

```diff
@@ -14,7 +17,8 @@ enum FormErrorCodeEnum {
   FORM_NOT_FOUND = 'FORM_NOT_FOUND',
   NO_FILE_UPLOADS = 'NO_FILE_UPLOADS',
   TOO_MANY_FILES = 'TOO_MANY_FILES',
-  FILES_TOO_BIG = 'FILES_TOO_BIG'
+  FILES_TOO_BIG = 'FILES_TOO_BIG',
+  STRIPE_CLIENT_ERROR = 'STRIPE_CLIENT_ERROR'
```

Comment:
We should keep `STRIPE_SCA_ERROR` in this list since it can be sent from the server.